.forge-snapshots/addLiquidity with empty hook.snap

```diff
@@ -1 +1 @@
-312159
\ No newline at end of file
+311322
```

Comment:
are these gas estimations?

Reply:
yes these are gas snapshots. you can see that we set them with `snapStart("<>")`